airbyte-commons-worker/src/main/java/io/airbyte/workers/helper/CatalogClientConverters.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
@xiaohansong did you reference any existing logic when adding this class? If so, it might make sense to add a comment pointing out where you referenced any existing logic from, just to make it slightly easier to maintain in case similar logic elsewhere ever changes

Reply:
Yep - was basically referring to the old CatalogConverter class but changing the class name from io.airbyte.api.model.generated.AirbyteCatalog to io.airbyte.api.client.model.generated.AirbyteCatalog (adding client); Will do.